docs/advanced/addons/pcidevices.md

```diff
@@ -94,4 +94,54 @@ We have also introduced a new filter to help you filter PCI devices by the under
 ![](/img/v1.2/pcidevices/SriovNetworkDevicesFilter.png)
 
 The newly created PCI device can be passed through to virtual machines like any other PCI device.
-![](/img/v1.2/pcidevices/SriovNetworkDevicesFilterResult.png)
\ No newline at end of file
+![](/img/v1.2/pcidevices/SriovNetworkDevicesFilterResult.png)
+
+## USB Devices
+
+_Available as of v1.4.0_
+
+A `USBDevice` in Harvester represents a USB device **on the node.** This is accomplished by using the `pcidevices-controller` addon.
+
+The devices can be passed through the hypervisor to a VM by creating a `USBDeviecClaim` resource, or by using the UI to enable passthrough. Passing a device through the hypervisor means that the VM can directly access the device, and effectively owns the device.
+
+It's slightly different from PCI Devices. For example, we have a USB Controller with four USB ports. We'll get a full control by creating `PCIDeviceClaim`. However, we can also get one USB port by creating `USBDeviceClaim`. The rest of USB ports are still available to the node.
+
+### Enabling Passthrough on a USB Device
+
+1. Now go to the `Advanced -> USB Devices` page:
+
+   ![](/img/v1.4/usbdevices/index.png)
+
+1. Search for your device by any keywords.
+
+   ![](/img/v1.4/usbdevices/search.png)
+
+1. Select the devices you want to enable for passthrough:
+
+   ![](/img/v1.4/usbdevices/select.png)
+
+1. Then click **Enable Passthrough** and read the warning message. If you still want to enable these devices, click **Enable** and wait for all devices to be `Enabled`.
+
+   ![](/img/v1.4/usbdevices/enable-done.png)
+
+### Attaching USB Devices to a VM
+
+After enabling these USB devices, you can navigate to the **Virtual Machines** page to create a new virtual machine or click **Edit Config** of existing virtual machine.
+
+Select **USB Devices** and use the **Available USB Devices** drop-down. Select the devices you want to attach from the list displayed and then click **Save**.
+
+![](/img/v1.4/usbdevices/attach-vm.png)
+
+### Using a passed-through USB Device inside the VM
+
+Boot the VM up, and run `lsusb` inside the VM, the attached USB devices will show up.
+
+![](/img/v1.4/usbdevices/usb-in-vm.png)
+
+### Limitations
+
+1. Don't support live migration since USB device is bound to the specified node.
+2. Don't support hot plug and replugging, please see [reference](https://github.com/kubevirt/kubevirt/issues/11979).
+3. Require re-creating a USBDeviceClaim to enable the USB device if device path changes in following situations:
+   - Re-plugging USB device.
+   - Rebooting the node.
```

Comment:
```suggestion
- Virtual machines with attached USB devices cannot be live-migrated because the devices are bound to a specific node.

- Hot-plugging and replugging of USB devices is not supported. For more information, see [KubeVirt Issue #11979](https://github.com/kubevirt/kubevirt/issues/11979).

- If the device path changes when you reattach the device or reboot the node, you must create a new USBDeviceClaim to enable the device.
```